proto/chirp.proto

```diff
@@ -13,6 +13,8 @@ message Message {
 	repeated TraceEntry trace = 6;
 	
 	bytes body = 3;
+
+	bool allow_recursive = 8;
```

Comment:
needs docs since this isn't immediately obvious why it's needed

Reply:
nit: don't mention specifically who needs it, keep it generalized and potentially use `cluster-server-scale` as an example instead of the root reason it exists.

Reply:
Is that not already how its structured? the first sentence is the general explanation and the second one is the example

Reply:
code-wise this is good to go. just a nit on the docs making it sound like this was designed specifically for that as opposed to being general purpose.

Reply:
The docs are what i am referring to, the first sentence is the general explanation